### ISSUE TITLE
the points num in deformable attention of RepDETR3D is 13， difficult to deploy

### ISSUE THREAD

**Issue by Mavericky-j** [exiawsh/StreamPETR#113]
Hello, thanks very much for your great work, and I wanna ask a question that have you tested the accuracy when num_pts equals 8 / 12 / 16? This kind of num_pts is much more friendly to deploy, especially in int8. looking for your reply, thanks very much!

**Comment by exiawsh** [comment]
@Mavericky-j Hi, thanks for your attention. I think 8 points are quite enough. Almost similar results. But you need to re-train the models.

**Comment by Mavericky-j** [comment]
Thanks for your reply in seconds and clear explanation ！I see, have a good day bro!

**Comment by Mavericky-j** [comment]
And it could be awesome if you release the model weights of 8 points, do you have a plan？looking for your reply~

**Comment by exiawsh** [comment]
@Mavericky-j I will do a ablation study about 4 and 8 points models. But I don't have enough GPU resources now. It will take some time.

**Comment by Mavericky-j** [comment]
I see, and hope you achieve GPU freedom as soon as possible~

**Comment by exiawsh** [comment]
@Mavericky-j I changed the num_points (13->4) in deformable attn and retrained the model. I found the results are good.
mAP: 0.5017                                                                                                                                                             
mATE: 0.5712
mASE: 0.2583
mAOE: 0.3877
mAVE: 0.2431
mAAE: 0.1954
NDS: 0.5853
Eval time: 312.7s

Per-class results:
Object Class            AP      ATE     ASE     AOE     AVE     AAE   
car                     0.684   0.383   0.144   0.060   0.236   0.192 
truck                   0.459   0.596   0.199   0.078   0.187   0.207 
bus                     0.532   0.664   0.199   0.128   0.422   0.266 
trailer                 0.278   0.908   0.229   0.536   0.152   0.142 
construction_vehicle    0.171   0.919   0.445   0.971   0.141   0.365 
pedestrian              0.560   0.584   0.281   0.411   0.286   0.136 
motorcycle              0.524   0.496   0.250   0.459   0.353   0.243 
bicycle                 0.494   0.484   0.248   0.685   0.168   0.012 
traffic_cone            0.678   0.318   0.301   nan     nan     nan   
barrier                 0.637   0.360   0.286   0.163   nan     nan   
I will update the config and model later.
